include/CL/cl2.hpp

```diff
@@ -18,7 +18,7 @@
  *
  *   \brief C++ bindings for OpenCL 1.0 (rev 48), OpenCL 1.1 (rev 33),
  *       OpenCL 1.2 (rev 15), OpenCL 2.0 (rev 29), OpenCL 2.1 (rev 17),
- *       and OpenCL 2.2 (V2.2-11).
+ *       OpenCL 2.2 (V2.2-11), and OpenCL 3.0 (V3.0.1-Provisional).
```

Comment:
Do we want to mention the exact tag? It'll likely get out of date.

Reply:
I'll just remove the tag for now.  It'll make OpenCL 3.0 look a little different than the prior OpenCL versions but maybe that's OK... and it's less error-prone anyhow.

Reply:
Now that the specs are unified, can't we just remove the all of tags and the "(rev XX)"s from all of the prior versions anyway?

Reply:
I'm fine with this.  I'll remove all the "(rev XX)" information.